client/castApp/src/testsock.c

```diff
@@ -100,10 +102,16 @@ static void testWakeup(void)
 
 MAIN(testsock)
 {
-    testPlan(18);
+    if (!getenv("SKIP_TEST_WAKEUP")) {
+        testPlan(18);
+    } else {
+        testPlan(14);
+    }
     osiSockAttach();;
     testUDP();
-    testWakeup();
+    if (!getenv("SKIP_TEST_WAKEUP")) {
+        testWakeup();
+    }
```

Comment:
I think that a better option is to use `testSkip`, something akin to
```
    testPlan(18);
    osiSockAttach();;
    testUDP();
    if (!getenv("SKIP_TEST_WAKEUP")) {
        testWakeup();
    } else {
        testSkip(4);
    }
```

Reply:
Even better (IMO) would be something like
```c
#ifdef __MACH__
    testSkip(4);
#else
    // everything else
#endif
```
or whatever the correct pre-processor directive is.